Jenkinsfile

```diff
@@ -72,39 +71,69 @@ spec:
          path: /var/run
 ''') {
     node(POD_LABEL) {
+        def GIT_BRANCH_NAME
```

Comment:
default to env.BRANCH_NAME and then you can get rid of the else section on line 79--81.

Reply:
I feel like the behaviour is more obvious if its all in the if/else. It shows its clearly not used for anything before the if/else that way